injective_functions/exchange/exchange.py

```
@@ -19,29 +19,28 @@ def __init__(self, chain_client) -> None:
         super().__init__(chain_client)
 
     async def get_subaccount_deposits(
-        self, subaccount_idx: int, denoms: str = None
+        self, subaccount_idx: int, denoms: List[str] = []
```

Ruff (0.7.0)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)